src/troute-network/troute/DataAssimilation.py

```diff
@@ -98,20 +98,42 @@ def __init__(self, network, from_files, value_dict, da_run=[]):
                     self._last_obs_df = _reindex_link_to_lake_id(lastobs_df, network.link_lake_crosswalk)
             
             else:
-                lastobs_file = streamflow_da_parameters.get("wrf_hydro_lastobs_file", None)
+                #TODO: Is it a sustainable to figure out if using NHD or HYfeature based on lastobs_crosswalk_file?
                 lastobs_crosswalk_file = streamflow_da_parameters.get("gage_segID_crosswalk_file", None)
-                lastobs_start = streamflow_da_parameters.get("wrf_hydro_lastobs_lead_time_relative_to_simulation_start_time", 0)
-                
-                if lastobs_file:
-                    self._last_obs_df = build_lastobs_df(
-                        lastobs_file,
-                        lastobs_crosswalk_file,
-                        lastobs_start,
-                    )
-                
+                if lastobs_crosswalk_file:
```

Comment:
I temporarily used whether lastobs_crosswalk_file exists or not to see if hydrofabric is NHD or HYfeature, but more sustainable way should be possible.

Reply:
It would be a good option but both generate exactly the same file type as nc and the same file name.